PROTO_tests/tests/setops_test.py

```diff
@@ -680,30 +680,52 @@ def test_index_of(self):
         # index of nan (reproducer from #3009)
         s = ak.Series(ak.array([1, 2, 3]), index=ak.array([1, 2, np.nan]))
         assert ak.indexof1d(ak.array([np.nan]), s.index.values).to_list() == [2]
+        rng = np.random.default_rng()
+        seeds = [rng.choice(2**63), rng.choice(2**63), rng.choice(2**63), rng.choice(2**63)]
+        print("seeds: \n", seeds)
 
+        def are_pdarrays_equal(pda1, pda2):
+            # we first check the sizes so that we won't hit shape mismatch
+            # before we can print the seed (due to short-circuiting)
+            return (pda1.size == pda2.size) and ((pda1 == pda2).all())
+
+        count = 0
         select_from_list = [
-            ak.randint(-(2**32), 2**32, 10),
+            ak.randint(-(2**32), 2**32, 10, seed=seeds[0]),
             ak.linspace(-(2**32), 2**32, 10),
-            ak.random_strings_uniform(1, 16, 10),
+            ak.random_strings_uniform(1, 16, 10, seed=seeds[1]),
         ]
         for select_from in select_from_list:
-            arr1 = select_from[ak.randint(0, select_from.size, 20)]
+            count += 1
```

Comment:
You increment count before using it in the loop, but then use it an additional time outside the loop, so I think you could use the same value twice.

Reply:
I think I see what your saying but for the time outside the loops it's `seeds[3]` not `seed[2]`, so I think having the same `count` value shouldn't matter as long as the overall seed is different